docs/backup-restore.md

```diff
@@ -23,3 +23,10 @@ One requirement for backup in Scalar DB on JDBC databases is that backups for al
 That means that you need to create a consistent snapshot by dumping all tables in a single transaction.
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
```

Comment:
BTW, Is this true even when we use ConsensusCommit?

Reply:
Thank you for reviewing this!

Yes. `mysqldump` command with `--single-transaction` and `pg_dump` command can take a backup that's automatically recoverable to a transactionally-consistent state. So, it's true even when we use ConsensusCommit.